presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestTransformSpatialPredicateToLeftJoin.java

```diff
@@ -221,6 +221,6 @@ public void testPushDownAnd()
 
     private RuleAssert assertRuleApplication()
     {
-        return tester().assertThat(new TransformSpatialPredicateToLeftJoin(tester().getMetadata()));
+        return tester().assertThat(new TransformSpatialPredicateToLeftJoin(tester().getMetadata(), null, null));
```

Comment:
pass `tester().getSplitManager(), tester().getPageSourceManager()` instead of `null`